server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -530,5 +530,6 @@ PICTURECHARSGROUP2: PICTURECharAcceptedOneTime+;
 WS2 : [ \t\f;]+ -> channel(HIDDEN);
 IS2: I S;
 LParIntegralRPar: LPARENCHAR INTEGERLITERAL RPARENCHAR;
-fragment PICTURECharAcceptedMultipleTime: ('0'|'9'|A|B|G|N|P|X|Z|PLUSCHAR|MINUSCHAR|SLASHCHAR|ASTERISKCHAR|COMMACHAR|DOLLARCHAR);
+fragment PICTUREPeriodAcceptables: ('0'|'9'|B|Z|CR|DB|ASTERISKCHAR|COMMACHAR|MINUSCHAR|PLUSCHAR|SLASHCHAR);
```

Comment:
took this from table present in below site
https://bs2manuals.ts.fujitsu.com/psCOBOL2000V16en/cobol2000-reference-manual-13781/data-division-cobol-compiler-reference-manual-120/data-description-entry-cobol-compiler-reference-manual-143/clauses-for-data-description-cobol-compiler-reference-manual-147/picture-clause-cobol-compiler-reference-manual-158

I am not sure , if this is same as IBM cobol.

Reply:
It would be better to ask Skott about it

Reply:
comment from scott:

Fujitsu supports a different COBOL syntax than IBM (similar, but there are differences).  we should be using the IBM syntax instead:

https://www.ibm.com/support/knowledgecenter/SS6SG3_4.2.0/com.ibm.entcobol.doc_4.2/PGandLR/igy3lr50.pdf

If you look at page 211 (PDF page 233), you will see how picture clauses are defined.

Reply:
![image](https://user-images.githubusercontent.com/69206564/99701617-f93d4280-2a94-11eb-8de2-79eb337309ae.png)
 This is the same as commited. So, I guess we can keep these.